articles/monitoring/guides/monitor-applications.md

```diff
@@ -13,20 +13,48 @@ useCase:
+You can monitor Auth0 using many approaches, each of them complementary with the others. Picking the approaches to use depends on your needs and investment possibilities.
 
-If you've extended Auth0 through [rules](/rules) or a [custom database connection](/connections/database/custom-db), you can build a synthetic transaction that exercises these capabilities using the [Resource Owner Password Grant](/api-auth/tutorials/password-grant). One way of doing this is to [Monitor Auth0 Using SCOM](/monitoring/guides/monitor-using-SCOM).
+## Synthetic transactions
+The simplest approach to monitor Auth0. This involves setting up a periodic request to perform an authentication transaction. If the request works successfully, Auth0 is working fine. If the request fails this _might_ indicate an issue with Auth0.
 
-Auth0 recommends using an authentication flow that doesn't require a user interface such as the **Resource Owner Password Grant**. That way, you can use a monitoring tool that doesn't have to mimick the actions of a user. Many monitoring services exist with this capability including:
+> There _might_ be an issue with Auth0, but the issue could also either be specific to the tenant used for the synthetic transaction or just a single failed request.
 
-* [New Relic](http://newrelic.com)
-* [Pingdom](http://pingdom.com)
+For synthetic transactions we recommend using setups that are close to your production tenant configuration as possible, potentially even using the same production tenant. Since setting up synthetic transactions with redirect flows and third party providers can be tricky, using the [Resource Owner Password Grant](/api-auth/tutorials/password-grant) is recommended. This flow neither involves browser redirects nor requires a UI.
 
-Use one of these services to execute synthetic authentication requests. 
+If you are using [rules](/rules) or [custom database connection(s)](/connections/database/custom-db) or other extensibility points, the synthetic transaction(s) should be configured to go through their logic.
 
-## Keep reading
+Tools like [Pingdom](http://pingdom.com) make setting up synthetic transactions a simple thing.
+
+### Check Period
+We recommend synthetic transactions happen with a periodicity of one minute. For this simple monitoring approach, that's a frequency that will not consume a lot of your Auth0 rate limit quota, while also providing timely responses if an issue were to be happening.
+
+### Limitations of synthetic transactions
+Synthetic transactions do not represent your end user's experience, but instead aim to provide a proxy metric for them. They might not be exercising the same flows your users are, they lack atomicity (typically run once a minute) and do not report on errors your end users might have seen. However, they are a simple and inexpensive way of getting good signal on the health of an Auth0 tenant.
```

Comment:
Synthetic transactions are a simple and inexpensive way monitoring an Auth0 tenant's health. However, they do have some limitations:

- Synthetic transactions do not represent your end user's experience. Instead they give you a proxy metric for them.
- Synthetic transactions might not use the same flows as your users
- They lack atomicity (typically run once a minute) and do not report on errors your end users might have seen.